build/build.proj

```diff
@@ -3,6 +3,7 @@
 
   <PropertyGroup>
     <Configuration>Debug</Configuration>
+    <Platform>&quot;Any CPU&quot;</Platform>
```

Comment:
:exclamation: It looks like this flag was lost during the migration from **build.ps1** to **build.proj**. The [previous build script](https://github.com/openstacknetsdk/openstack.net/blob/8ad5dabe635942f9013705193c2f95331b1ce905/build/build.ps1#L112) used the **Mixed Platforms** platform:

```xml
<Platform>Mixed Platforms</Platform>
```